libcyphal/include/libcyphal/transport/ip/v4/address.hpp

```diff
@@ -41,7 +48,7 @@ class Address final
     {
     }
 
-    /// @brief Explicit Parameter Constructor as classical four part address
+    /// Explicit Parameter Constructor as classical four part address
```

Comment:
a lot of style changes at the front. Only substantive changes are the new static methods at the end.